website/azurerm.erb

```diff
@@ -174,6 +174,15 @@
               </ul>
             </li>
 
+            <li<%= sidebar_current("docs-azurerm-resource-image") %>>
+              <a href="#">Image Resources</a>
```

Comment:
is it worth placing this in the storage sub-section?

_related:_ I think we should probably reorganise the sidebar in general